### ISSUE TITLE
`rvm install 2.7.8` fails on Debian 12 (bookworm) with master (`45caf95`)

### ISSUE THREAD

**Issue by georgeraraujo** [rvm/rvm#5372]
## Description

`rvm install 2.7.8` fails on Debian 12 (bookworm)

## Steps to reproduce

 1. Install Debian 12
 2. `gpg --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB`
 3. `\curl -sSL https://get.rvm.io | bash`
 4. `rvm requirements`
 5. `rvm install 2.7.8`

## Expected behavior

Ruby 2.7.8 builds and installs sucessfully.

## Actual behavior

* [command](https://gist.github.com/georgeraraujo/cb849b3b8eb2c68e0f103aaa94dd978d)
* [config.log](https://gist.github.com/georgeraraujo/5f43de00b11aa4d98ae3c36eab0f3ce7)
* [make.log](https://gist.github.com/georgeraraujo/0d72a2ee3abd9e0e868365524980bab1)

## Environment info

```
vboxuser@debian12:~$ rvm info

system:

  system:
    uname:        "Linux debian12 6.1.0-10-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.38-1 (2023-07-14) x86_64 GNU/Linux"
    name:         "Debian"
    version:      "12"
    architecture: "x86_64"
    bash:         "/usr/bin/bash => GNU bash, version 5.2.15(1)-release (x86_64-pc-linux-gnu)"
    zsh:          " => not installed"
    remote_path:  "debian/12/x86_64"

  rvm:
    version:      "1.29.12-next (master)"
    updated:      "31 minutes 20 seconds ago"
    path:         "/home/vboxuser/.rvm"
    autolibs:     "[4] Allow RVM to use package manager if found, install missing dependencies, install package manager (only OS X)."

  homes:
    gem:          "not set"
    ruby:         "not set"

  binaries:
    ruby:         ""
    irb:          ""
    gem:          ""
    rake:         ""

  environment:
    PATH:         "/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/home/vboxuser/.rvm/bin:/home/vboxuser/.rvm/bin"
    GEM_HOME:     ""
    GEM_PATH:     ""
    MY_RUBY_HOME: ""
    IRBRC:        ""
    RUBYOPT:      ""
    gemset:       ""


vboxuser@debian12:~$
```

**Comment by raelgc** [comment]
My guess is that Debian will face the same issue as Ubuntu: new version is shipped only with openssl-3 and ruby 2.6 up to 3.0 will request openssl-1.

Try the solution described on this comment: https://github.com/rvm/rvm/issues/5209#issuecomment-1264562996

**Comment by georgeraraujo** [comment]
It worked! THANKS!

```
vboxuser@debian12:~$ rvm install 2.7.8 --with-openssl-dir=$HOME/.openssl/openssl-1.1.1u
Checking requirements for debian.
Requirements installation successful.
Installing Ruby from source to: /home/vboxuser/.rvm/rubies/ruby-2.7.8, this may take a while depending on your cpu(s)...
ruby-2.7.8 - #downloading ruby-2.7.8, this may take a while depending on your connection...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 14.1M  100 14.1M    0     0  1185k      0  0:00:12  0:00:12 --:--:-- 1471k
ruby-2.7.8 - #extracting ruby-2.7.8 to /home/vboxuser/.rvm/src/ruby-2.7.8.....
ruby-2.7.8 - #configuring.............................................................
ruby-2.7.8 - #post-configuration..
ruby-2.7.8 - #compiling............................................................................................
ruby-2.7.8 - #installing.................
ruby-2.7.8 - #making binaries executable...
Installed rubygems 3.1.6 is newer than 3.0.9 provided with installed ruby, skipping installation, use --force to force installation.
ruby-2.7.8 - #gemset created /home/vboxuser/.rvm/gems/ruby-2.7.8@global
ruby-2.7.8 - #importing gemset /home/vboxuser/.rvm/gemsets/global.gems..........................................................
ruby-2.7.8 - #generating global wrappers........
ruby-2.7.8 - #gemset created /home/vboxuser/.rvm/gems/ruby-2.7.8
ruby-2.7.8 - #importing gemsetfile /home/vboxuser/.rvm/gemsets/default.gems evaluated to empty gem list
ruby-2.7.8 - #generating default wrappers........
ruby-2.7.8 - #adjusting #shebangs for (gem irb erb ri rdoc testrb rake).
Install of ruby-2.7.8 - #complete
Ruby was built without documentation, to build it run: rvm docs generate-ri
vboxuser@debian12:~$ rvm use 2.7.8
Using /home/vboxuser/.rvm/gems/ruby-2.7.8
vboxuser@debian12:~$ ruby -v
ruby 2.7.8p225 (2023-03-30 revision 1f4d455848) [x86_64-linux]
vboxuser@debian12:~$
```

Is the following message something to worry about?

```
Installed rubygems 3.1.6 is newer than 3.0.9 provided with installed ruby, skipping installation, use --force to force installation.
```
